src/pages/category/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { useGetCategoriesQuery } from '@/queries/category';
+import CategoryItem from './components/category-item';
+import { gridWrapper, wrapper, wrapperMenu, hr } from './index.styles';
+
+export default function Category() {
+  const { data } = useGetCategoriesQuery();
+  if (!data) return null;
+
+  const categories = data.data;
+
+  const categoryItems: React.ReactNode[] = [];
+  for (let i = 1; i < categories.length; i++) {
+    const categoryItem = categories[i];
+    categoryItems.push(<CategoryItem key={categoryItem.id} item={categoryItem} />);
+  }
+
+  return (
+    <div css={wrapper}>
+      <div css={wrapperMenu}>
+        <h3>전체 메뉴</h3>
+        <hr css={hr} />
+        <div css={gridWrapper}>{categoryItems}</div>
+      </div>
+    </div>
+  );
```

Comment:
`for`문을 직접 쓰는 것보다 `map`과 같은 배열 메서드를 더 많이 쓰는 것 같아요!
아래처럼 작성하면 더욱 간결한 코드가 될 것 같습니다!
```suggestion
  return (
    <div css={wrapper}>
      <div css={wrapperMenu}>
        <h3>전체 메뉴</h3>
        <hr css={hr} />
        <div css={gridWrapper}>
          {categories?.map((category) => (
            <CategoryItem key={categoryItem.id} item={categoryItem} />
          ))}
        </div>
      </div>
    </div>
  );
```

Reply:
수정하였습니다.